container_images/cli-tools-module-tests/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM gcr.io/gcp-guest/gotest
+
+COPY gcsfuse-apt-key.gpg .
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -q -y qemu-utils gnupg ca-certificates \
```

Comment:
is there a benefit to having this all as one long && list?

Reply:
Each `RUN` command creates a new layer in the final image. Here's a good explanation: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run